src/panels/UserPanel.php

```
@@ -124,18 +123,25 @@ public function canSwitchUser(): bool
 
-        if (!is_array($controller) || !$controller[0] instanceof UserController) {
+        if (is_array($actionConfig)) {
+            $actionConfig = $actionConfig['class'] ?? null;
+        }
+
+        if (!is_string($actionConfig) || !class_exists($actionConfig)) {
             return false;
         }
 
-        $action = $controller[0]->createAction('set-identity');
+        $action = Yii::createObject($actionConfig);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'canSwitchUser'. (undefined)

(StaticAccess)